include/zephyr/bluetooth/att.h

```diff
@@ -91,6 +91,12 @@ size_t bt_eatt_count(struct bt_conn *conn);
 
 #endif /* CONFIG_BT_EATT */
 
+enum bt_att_chan_preference {
```

Comment:
Looks much better now, thanks! However, for consistency, please also use the shorter `pref` here, i.e. `bt_att_chan_pref`